.github/workflows/backstage-release.yaml

```diff
@@ -0,0 +1,72 @@
+name: Publish release on NPM
+
+on:
+  # When manually triggered
+  workflow_dispatch:
+  # When there's a release created
+  release:
+    types: [ published ]
+
+jobs:
+  publish-release-on-npm:
+    name: Publish release on NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'backstage/.nvmrc'
+          cache: 'yarn'
+          cache-dependency-path: 'backstage/yarn.lock'
+
+      - name: Print environment
+        run: |
+          node --version
+          yarn --version
+
+      - name: Print release name
+        run: |
+          echo "Release name: ${{ github.event.release.name }}"
+
+      - name: Install dependencies
+        shell: bash
+        working-directory: ./backstage
+        run: yarn --prefer-offline --frozen-lockfile
+
+      - name: Install tooling
+        shell: bash
+        working-directory: ./backstage
+        run: npm install @backstage/cli -g
+
+      - name: Run tests
+        shell: bash
+        working-directory: ./backstage
+        run: |
+          export PATH="./node_modules/.bin/:$PATH"
+          yarn test
```

Comment:
Running tests here seems a good additional check, however, I think we need to run the tests _before_ Prow publishes a release in the Prow jobs

Reply:
Created https://github.com/knative-extensions/backstage-plugins/issues/21